docs/sdkx_android/users.mdx

```diff
@@ -119,14 +119,20 @@ You need to be aware of some considerations when using the `Helpshift.login()` A
 - It is best to call login immediately when your app user logs in.
 - If you are setting `fullPrivacy` to true, you shouldn't use email as the only identifier in the login API for that user. This will result in the creation of an anonymous user.
 
+<Admonition type="info" title="Note">
+
+From 10.3.0, `Helpshift.login()` API will return a boolean indicating wheather login was successful or not.This change is in exisiting API to introduce mandatory validations for login parameters and hence this might affect code compilation.
```

Comment:
```suggestion
From 10.3.0, `Helpshift.login()` API will return a boolean indicating wheather login was successful or not. This change is in existing API to introduce mandatory validations for login parameters and hence this might affect code compilation.
```